flash/image/segmentation/data.py

```diff
@@ -148,7 +149,7 @@ def load_sample(self, sample: Mapping[str, Any]) -> Mapping[str, Union[torch.Ten
         img_labels_path = sample[DefaultDataKeys.TARGET]
 
         # load images directly to torch tensors
-        img: torch.Tensor = torchvision.io.read_image(img_path)  # CxHxW
+        img: torch.Tensor = FT.to_tensor(default_loader(img_path))  # CxHxW
```

Comment:
Would maybe be better to just use `default_loader` here and then convert to tensor in the preprocess `to_tensor_transform`. Could make a good follow up PR